test/simplifyExpression/basicsSearch/basicTest.js

```diff
@@ -87,7 +85,7 @@ describe('basic rules', function() {
       ['0 + x', 'x'],
       ['0 + (x + 1)', 'x + 1'],
     ];
-    tests.forEach(t => testSimplify(t[0], t[1], basic_rules.removeAdditionOfZero));
+    tests.forEach(t => testSimplify(t[0], t[1], basics.REMOVE_ADDING_ZERO, ChangeTypes.REMOVE_ADDING_ZERO, basics.REMOVE_ADDING_ZERO_REVERSE));
```

Comment:
Yeah, I'm not too satisfied with it myself. We can make it cleaner once we have the array parameter working.